rotkehlchen/api/v1/schemas.py

```diff
@@ -2318,3 +2314,14 @@ def validate_schema(  # pylint: disable=no-self-use
                 message='Can\'t delete the etherscan node',
                 field_name='identifier',
             )
+
+
+class OptionalAddressesListSchema(AsyncQueryArgumentSchema):
```

Comment:
Do this as I explained above
```suggestion
class OptionalAddressesListSchema(Schema):
```